### PR TITLE
Popup Disign

### PR DESCRIPTION
**Pull request by Romlion**

### CODE REVIEW

File: frontend/src/components/LevelPage/TargetButton/TargetButton.scss [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+.Ptéro {
+  width: 10vw;
+  height: 10vw;
+  z-index: 1;
+}
+
+.Ptéro-egg {
+  width: 2vw;
```
Comment (ItsMatDev) [inline frontend/src/components/LevelPage/TargetButton/TargetButton.scss:31]: vh

File: frontend/src/components/LevelPage/TargetButton/TargetButton.scss [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  }
+}
+
+.buttonPtetodactyle {
+  all: unset;
+}
+
+.Ptéro {
+  width: 10vw;
+  height: 10vw;
```
Comment (ItsMatDev) [inline frontend/src/components/LevelPage/TargetButton/TargetButton.scss]: vh

File: frontend/src/components/LevelPage/TargetButton/TargetButton.scss [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  z-index: 0;
+}
+
+.popPtéro {
+  position: absolute;
+  background-color: $primaryColor;
+  border: 2px solid $secondaryColor;
+  border-radius: 50px;
+  width: 30vw;
+  height: 15vw;
```
Comment (ItsMatDev) [inline frontend/src/components/LevelPage/TargetButton/TargetButton.scss]: vh

**Review by ItsMatDev — CHANGES_REQUESTED** [review]

**Review by ItsMatDev — APPROVED** [review]

File: frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.jsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+import Brain from "../../../../assets/pictures/Brain.png";
+import Egg from "../../../../assets/pictures/jurassic/Pterodactyle-egg.png";
+
+import "./TargetButton.scss";
+
+function TargetButton() {
+  const [popupMessage, setPopupMessage] = useState("");
+  function HandleEvent() {
+    setPopupMessage(
+      <article className="popPtéro">
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.jsx]: No French in code

File: frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.jsx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        <img src={Pterodactyle} alt="Ptéro" className="Ptéro" />
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.jsx]: No French in code

File: frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.jsx [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        >
+          close
+        </button>
+        <button
+          type="button"
+          onKeyDown={HandleEvent}
+          onClick={HandleEvent}
+          className="buttonEgg"
+        >
+          <img src={Egg} alt="oeuf" className="Ptéro-egg" />
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.jsx]: No French in code

File: frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.jsx [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+  }
+  return (
+    <>
+      <button
+        type="button"
+        onKeyDown={HandleEvent}
+        onClick={HandleEvent}
+        className="buttonPtetodactyle"
+      >
+        <img src={Pterodactyle} alt="pterodactyle" className="Pterodactyle" />
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.jsx]: No PascalCase in CSS

Comment (Yoann-Fortin) [reply frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.jsx]: Is the alt value in English?

File: frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.scss [hunk 8]
```diff
@@ -0,0 +1,82 @@
+@import "../variables.scss";
+
+.close {
+  position: absolute;
+  right: 2rem;
+  top: 0.5rem;
+}
+
+.Pterodactyle {
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.scss]: No PascalCase in CSS

File: frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.scss [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+    filter: drop-shadow(3px 5px 2px rgb(0 0 0 / 0.2));
+    cursor: pointer;
+  }
+}
+
+.buttonPtetodactyle {
+  all: unset;
+}
+
+.Ptéro {
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.scss]: No French and PascalCase in CSS

File: frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.scss [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+  width: 10vw;
+  height: 10vw;
+  z-index: 1;
+}
+
+.Ptéro-egg {
+  width: 2vw;
+  height: 2vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.scss]: No vw for a height

File: frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.scss [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+  z-index: 0;
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.scss]: The default z-index value is 0, which is what you might call dead code.

File: frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.scss [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+.popPtéro {
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.scss]: No French in code

File: frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.scss [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+  position: absolute;
+  background-color: $primaryColor;
+  border: 2px solid $secondaryColor;
+  border-radius: 50px;
+  width: 30vw;
+  height: 15vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.scss]: No vw for a height

File: frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.scss [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+.name {
+  font-size: 3vw;
+}
+
+.info {
+  font-size: 0.5vw;
+  flex: 1;
+  border-left: 2px solid $secondaryColor;
+  border-right: 2px solid $secondaryColor;
+  height: 1.5vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/TargetButton/TargetButton.scss]: No vw for a height

**Review by Yoann-Fortin — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**